### ISSUE TITLE
contentful-import only parts

### ISSUE THREAD

**Issue by aditya-prasad-projects** [datocms/js-datocms-client#66]
I have an contenful account, wanted to migrate to datocms, but not everything, nor do I want to delete content in contentful. So, wanted to implement or provide a new option while running dato contentful-import, that takes in content types.

Not sure if this issue was discussed and not implemented

**Comment by matjack1** [comment]
Hello @aditya-prasad-projects!

First, we don't delete content on Contentful when importing, so you should be safe there.

Then have you thought about importing everything and then deleting the models that you don't need? It's a lot of content?

If you want to try adding that feature I'm more than happy to review the code and test it myself! Feel free to be in touch either here or on https://community.datocms.com/

**Comment by aditya-prasad-projects** [comment]
I took a look at how it can be done, I came to the same conclusion as you suggested, but instead of deleting what was imported, I had thought I will just skip the rest and just migrate whatever the user specified, looking at command.js now to see which all functions I would have to change to just migrate what the user wants.
But thinking about it now, makes more sense to just delete it, rather than filter out every time.

I had a question though, can i change the parameters passed to command.js and what command.js passes to the functions its calling? I had a doubt, if there are some guarantees that the project provides that I need to be aware of?
